### PR TITLE
Fail tests on unexpected console warning / error

### PR DESCRIPTION
**Pull request by KaylaBrady**
ticket: https://app.asana.com/0/1152340551558956/1203458460363782/f

See examples of what the failing test output will look like [here](https://github.com/mbta/skate/actions/runs/4441085829/jobs/7795718019).

### CODE REVIEW

**Review by lemald — APPROVED** [review]
Awesome!

File: assets/package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
     "file-loader": "^6.2.0",
     "fishery": "^2.2.2",
     "jest": "^26.6.3",
+    "jest-fail-on-console": "^3.0.2",
```
Comment (firestack) [inline assets/package.json:75]: note(non-blocking): It looks like [`3.1.0`](https://www.npmjs.com/package/jest-fail-on-console/v/3.1.0) was posted 4 days ago! can we use that?

**Review by firestack — APPROVED** [review]
Yay! glad to have this!
one note about the version of the package but otherwise LGTM!

File: assets/package.json (hunk 1 above)
Comment (KaylaBrady) [reply assets/package.json]: Thanks for noticing, just updated!

**Pull request merged**